README.md

```diff
@@ -208,6 +216,76 @@ If at least one `<expression>` does not fit onto the line, *all* lines are wrapp
     <expression>)
 ```
 
+## Loops
+
+Conditions in loops are formatted equally to conditions in if expressions.
+
+### While loops
+
+```Ada
+while <condition> loop
+   <statements>
+end loop;
+```
+
+With multi line conditions:
+
+```Ada
+while
+   (((<condition>
+     and then <condition>)
+    or else <condition>)
+   and then <condition>)
+loop
+   <statements>
+end loop;
+```
+
+### For loops
+
+```Ada
+for <index> in <range> loop
+   <statements>
+end loop;
+
+for <element> of <array> loop
+   <statements>
+end loop;
+```
+
+For long ranges and arrays that do not fit on a line:
+
+```Ada
+for <index> in
+   First
+   .. Last
+loop
+   <statements>
+end loop;
```

Comment:
Should we clarify that we put `First` and `Last` on a single line each, even if `First .. Last` fit on a single line?

Reply:
This is a good question. I'd prefer to put `First .. Last` on a single line if they would fit. However if they're too long for the first line they might be long enough to be split anyway.

Reply:
For the sake of simplicity I'd stick with separate first/last (as with conditionals - when they don't fit onto one line we split them).

Reply:
> I'd prefer to put First .. Last on a single line if they would fit.

+1

Reply:
I added further examples that allow both a single line and two lines in for loop conditions.